streampipes-client-go/streampipes/data_lake_measure_api.go

```diff
@@ -0,0 +1,180 @@
+//
+// Licensed to the Apache Software Foundation (ASF) under one or more
+// contributor license agreements.  See the NOTICE file distributed with
+// this work for additional information regarding copyright ownership.
+// The ASF licenses this file to You under the Apache License, Version 2.0
+// (the "License"); you may not use this file except in compliance with
+// the License.  You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package streampipes
+
+import (
+	"github.com/apache/streampipes/streampipes-client-go/streampipes/config"
+	"github.com/apache/streampipes/streampipes-client-go/streampipes/internal/serializer"
+	"github.com/apache/streampipes/streampipes-client-go/streampipes/internal/util"
+	"github.com/apache/streampipes/streampipes-client-go/streampipes/model/data_lake"
+	"io"
+	"log"
+	"net/http"
+)
+
+// DataLakeMeasure connects to the DataLakeMeasure endpoint of streamPipes.
+// DataLakeMeasure supports GET and DELETE to delete or obtain resources
+// The specific interaction behavior is provided by the method bound to the DataLakeMeasure struct.
+type DataLakeMeasure struct {
+	endpoint
+}
+
+func NewDataLakeMeasures(clientConfig config.StreamPipesClientConfig) *DataLakeMeasure {
+	// NewDataLakeMeasure is used to return an instance of *DataLakeMeasure,
+
+	return &DataLakeMeasure{
+		endpoint{config: clientConfig},
+	}
+}
+
+func (d *DataLakeMeasure) AllDataLakeMeasure() ([]data_lake.DataLakeMeasure, error) {
+	// Get a list of all measure
```

Comment:
You can move this comment above the method name, so it can serve as the documentation for this method.